codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -26,6 +26,12 @@ customising->customizing
 dialogue->dialog
 digitise->digitize
 digitising->digitizing
+discretisation->discretization
+discretise->discretize
+discretised->discretized
+diskretisation->discretization
+diskretise->discretize
+diskretised->discretized
```

Comment:
These can be removed from here, this dictionary should only be valid British words mapping to valid American ones, e.g. colour->color, Travis checks that both left and right of these dictionary entries are in the dictionary, hence it's complaining that diskretisation isn't a real word (because it isnt').